packages/ui/src/components/Dropdown/index.tsx

```diff
@@ -1,3 +1,4 @@
+"use client";
```

Comment:
I added directives like this in most UI components that are not RSC-compatible, but I probably missed some.

If a component doesn't have this directive while being used in RSC in Hub, Hub's build will fail.

What's worse, if an RSC in Hub imports the entire `@quri/ui` package, Hub's build will fail (Next.js bundler is not very smart). That's how I triggered the condition that this was necessary, but then I've fixed it in Hub's source in a different way, so this directive doesn't matter for now.